### ISSUE TITLE
CommentAPI namespace support

### ISSUE THREAD

**Issue by mpgirro** [mpgirro/stalla#87]
The CommentAPI namespace is used to expose the URL endpoint for posting comments. The URI is `http://wellformedweb.org/CommentAPI/`. 24,6% of feeds declare the namespace, so it's pretty common.

The wellformedweb.org domain is down, and the namespace is otherwise not very well documented, but I still found some info. The wayback machine shows the [original spec](https://web.archive.org/web/20080701185821/http://wellformedweb.org/story/9). The only element it defines is the `wfw:comment` element in the `<item>`, but it's used in < 0,1% of feeds. Back in 2003, [Chris Sells](http://sellsbrothers.com) proposed an [additional element](http://sellsbrothers.com/12548) `wfw:commentRss`, which is used in 8,4% of feeds (also < 0,1% use the wrong `wfw:commentRSS`).

Chris Sells also mentions that his element is for an item-level comment endpoint URL, but as I understand the CommentAPI spec, the `wfw:comment` basically does the same. Right now I'm a bit confused here what the actual difference between the two elements is.

Example: 

```
<wfw:comment xmlns:wfw="http://wellformedweb.org/CommentAPI/">
  http://bitworking.org/news/comments/52
</wfw:comment>
<wfw:commentRss xmlns:wfw="http://wellformedweb.org/CommentAPI/"> 
  http://bitworking.org/news/commentsRss/52
</wfw:commentRss>
```

**Comment by rock3r** [comment]
For what I understand, the difference is:
 * `wfw:comment` points to a REST endpoint that you can POST to, in order to _submit_ comments for a certain `item`
 * `wfw:commentRss` points to a secondary RSS feed that _serves_ existing comments for a certain `item`

This means, they are complimentary. It would make sense to implement them both, while we're at it — they're pretty simple, at any rate — although I completely understand why many more feeds would expose the "read" URL tag (`wfw:commentRss`) but not the "write" URL tag (`wfw:comment`): for readers and aggregators that's much more useful.

**Comment by mpgirro** [comment]
You are totally right, should have read the third paragraph and not just skip over the HTTP examples 😅.

I'd definitely implement them both. In general, a frequency < 0,1% does not discourage me from supporting something. If there is no parser/writer support for something, almost nobody will use a standard, and I'd like to change that :)
